lib/msal-browser/src/error/BrowserAuthError.ts

```diff
@@ -156,6 +156,10 @@ export const BrowserAuthErrorMessage = {
     databaseUnavailable: {
         code: "database_unavailable",
         desc: "IndexedDB, which is required for persistent cryptographic key storage, is unavailable. This may be caused by browser privacy features which block persistent storage in third-party contexts."
+    },
+    keyGenerationFailed: {
+        code: "key_generation_failed",
+        desc: "Failed to generate cryptographic key"
```

Comment:
Anything else we can provide here?

Reply:
Added a details variable to the error creation method to get some more info from the failed calls to generateKeyPair

Reply:
I dont see that update?

Reply:
Ah nvm, github wasn't showing it to me.